### DISCUSSION TITLE
Consider adding option for not downloading overlays

### DISCUSSION THREAD

**Discussion by Smiths76245** [kubilus1/autoortho#322] [Ideas]
Hi,

Thank you for making this great program. For users of Simheaven sceneries, we don't require the overlays and so if you could consider making this optional via GUI or config that would be really appreciated. Thanks.

**Comment by kubilus1** [comment]
Yeah, this is actually in the works, in fact.  The latest version will give me more flexibility in how scenery packages are built out.  Currently they are intertwined, but in the future I'll likely separate these.

**Comment by marloscaida** [comment]
Kubilus1: Thank you very much for your great work.
It really totally changes the dimension of X-Plane.
A small contribution to the project would be the following:
Every time I install a new version it offers me another new version of SCENERY (in my case South America Ortho Set) since it doesn't detect that I have the latest version (0.0.53) installed. Something would have to be done to detect it as already installed and not to go through the process again.
This time I didn't install them and AO 0.5.1-3 works just as well.
thank you very much again
Regards
carlos

**Reply by kubilus1** [reply]
Thanks, I'm happy to hear that it is useful to you.

Yeah between version 0.4.x and 0.5.x I had to update a bit on how scenery is seen and managed.  I don't plan on any major changes to scenery layout going forward so this *should* be stable.

Of course it will still show up if there is an update to the scenery available.   But if it is working then you should be all set.
